packages/hiro-graph-orm-mappings/src/knowledge-item.js

```diff
@@ -0,0 +1,36 @@
+/**
+ *  A Knowledge Item
+ */
+export default {
+    name: "KnowledgeItem",
+    ogit: "ogit/Automation/KnowledgeItem",
+    required: {
+        body: "ogit/Automation/knowledgeItemFormalRepresentation",
+        deployed: {
```

Comment:
there is also `isDeployed`, maybe this one should be `deploy` or `deployToEngine`